### PR TITLE
ChangeLog updates for core modules

### PR DESCRIPTION
**Pull request by sima-zhu**

### CODE REVIEW

File: sdk/core/azure-core-amqp/CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+- Renamed `getHost` to `getHostName`.
+- Cleanup and introduced OpenCensus Tracing plugin.
+- Added `PROTON_IO` in ErrorCondition.
+- Added `ProxyConfiguration` for API `createConnectionHandler`.
+
+## Version 1.0.0-preview.4 (2019-09-09)
+- Support tracing for azure-core-amqp.
+
+## Version 1.0.0-preview.3 (2019-08-05)
+- Make ClientLogger thread-safe.
```
Comment (conniey) [inline sdk/core/azure-core-amqp/CHANGELOG.md]: There were no updates in ClientLogger on our end.

File: sdk/core/azure-core-amqp/CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+- Fixes authorization when using client credentials.
+- Changed FullyQualifiedDomainName -> FullyQualifiedNamespace.
+- Renamed `BatchOptions` -> `CreateBatchOptions` and added `getRetryMode`.
+- Renamed `ProxyConfiguration` -> `ProxyOption`s.
+- Removed cloneable from retry policies.
+- Renamed `RetryOptions`, `RetryPolicy` -> `AmqpRetryOptions`, `AmqpRetryPolicy`.
+- Updated `RetryMode` -> `AmqpRetryMode`.
+- Updated CBS -> ClaimsBasedSecurityNode.
+- Removed final from RetryPolicy.
+- Renamed from getProperties() -> getEventHubProperties().
```
Comment (conniey) [inline sdk/core/azure-core-amqp/CHANGELOG.md]: This is in our actual azure-messaging-eventhubs package.

Comment (conniey) [reply sdk/core/azure-core-amqp/CHANGELOG.md]: Same with changes in line 15 and 16

File: sdk/core/azure-core-http-okhttp/CHANGELOG.md [hunk 3]
```diff
@@ -0,0 +1,14 @@
+# Release History
+
+## 1.2.0-beta.1 (Unreleased)
+
+## Version 1.1.0 (2019-11-26)
+- Switch to JUnit version 5.
```
Comment (hemanttanwar) [inline sdk/core/azure-core-http-okhttp/CHANGELOG.md]: I do not see okhttp 1.1.0  released yet http://repo2.maven.apache.org/maven2/com/azure/azure-core-http-okhttp/

**Pull request merged**